src/lang/haw.js

```diff
@@ -17,47 +17,47 @@ export default {
-	"components:gradeResult:overallGrade": "Papa Laulā",
-	"components:gradeResult:studentGradePreviewLabel": "Nānā Haʻawina o ka Papa",
-	"components:gradeResult:studentGradePreviewNotShown": "ʻAʻole hōʻike ʻia ka papa i nā haumāna",
+	"components:gradeResult:cannotBeNegative": "(ʻaʻole hiki ke emi iho ma mua o 0)",
+	"components:gradeResult:clearManualOverride": "Holoi Ka Hoʻopololei Kauoha",
+	"components:gradeResult:gradeScoreLabel": "Nā Kaha helu",
+	"components:gradeResult:manuallyOverrideGrade": "Hoʻopau lima i ke kaha lima",
+	"components:gradeResult:numeratorOutOfDenominator": "{numerator} o ka {denominator}",
+	"components:gradeResult:outOfDenominator": "waho o ka {denominator}",
+	"components:gradeResult:overallGrade": "Nā Kaha Laulā",
+	"components:gradeResult:studentGradePreviewLabel": "Nānā Nā Kaha o ka haumāna",
+	"components:gradeResult:studentGradePreviewNotShown": "ʻAʻole hōʻike ʻia ka Nā Kaha i nā haumāna",
 	"components:mediaPlayer:captions": "Nā huaʻōlelo",
 	"components:mediaPlayer:captionsVtt": "Nā huaʻōlelo (.vtt)",
 	"components:mediaPlayer:closedCaptionsAcronym": "CC",
 	"components:mediaPlayer:default": "Default",
 	"components:mediaPlayer:download": "Hoʻoiho",
-	"components:mediaPlayer:exitFullscreen": "Haʻalele i ka pale piha",
+	"components:mediaPlayer:exitFullscreen": "Haʻalele i ka Pale Hoʻokahi",
 	"components:mediaPlayer:fullscreen": "Palena piha",
 	"components:mediaPlayer:language": "ʻŌlelo",
-	"components:mediaPlayer:loadErrorMessage": "Auē, pilikia ka hoʻouka ʻana i kēia faila media",
+	"components:mediaPlayer:loadErrorMessage": "Auē, pilikia ka hoʻouka ʻana i kēia faila pāpaho",
 	"components:mediaPlayer:loadSuccessMessage": "Ua hoʻouka maikaʻi ka media",
 	"components:mediaPlayer:menu": "papa kuhikuhi",
-	"components:mediaPlayer:mute": "Kamaʻilio",
-	"components:mediaPlayer:off": "Paʻa",
+	"components:mediaPlayer:mute": "Mute",
+	"components:mediaPlayer:off": "Mauʻu",
 	"components:mediaPlayer:pause": "Hoʻomaha",
 	"components:mediaPlayer:play": "Pāʻani",
-	"components:mediaPlayer:playbackSpeed": "ʻO ka māmā pāʻani",
+	"components:mediaPlayer:playbackSpeed": "ʻO ka ʻāpono o ka holo ʻana",
 	"components:mediaPlayer:quality": "ʻAno maikaʻi",
-	"components:mediaPlayer:retry": "E hoao hou",
+	"components:mediaPlayer:retry": "Hoʻāʻo hou",
 	"components:mediaPlayer:searchPlaceholder": "Huli...",
-	"components:mediaPlayer:seekSlider": "imi pahee",
-	"components:mediaPlayer:settings": "hoʻonohonoho",
-	"components:mediaPlayer:showSearchInput": "Hōʻike i ka hoʻokomo ʻimi",
-	"components:mediaPlayer:sliderBarProgress": "ka holomua ʻana o ka pahu slider",
-	"components:mediaPlayer:speed": "ka māmā holo",
+	"components:mediaPlayer:seekSlider": "seek slider",
+	"components:mediaPlayer:settings": "settings",
+	"components:mediaPlayer:showSearchInput": "Show search input",
+	"components:mediaPlayer:sliderBarProgress": "slider bar progress",
+	"components:mediaPlayer:speed": "speed",
```

Comment:
These are going back to English... 😕

Reply:
What the... but just 5 of them...

Reply:
Good catch -- I've reverted to the original values in `main` and applied that here as well.